tests/testthat/test-join_ald_scenario.R

```diff
@@ -119,18 +118,18 @@ test_that("include/excludes `plant_location`s inside/outside a region", {
   expect_false(any(unique(out$plant_location) %in% "us"))
 })
 
-test_that("case insensitive to input `plant_location`", {
+test_that("is case-insensitive to `plant_location` inputs", {
   out1 <- join_ald_scenario(
     fake_matched(),
     ald = fake_ald(plant_location = c("de")),
-    scenario = fake_scenario(region = "oecd_europe"),
+    scenario = fake_scenario(region = "oecd"),
```

Comment:
@jdhoffa,

Sorry for pushing this PR in such a messy state. I'll cleanup later. Now I just  wanted to show you that this seems weird. 

* This and a few other tests outpus a 0-row tibble
* "oecd_europe" is no longer a valid value;  our `region*" datasets intead have "oecd" (no "_europe" suffix).

I'll keep exploring but can you immedaitely see what's going on?

(BTW, I'm glad the warning exposed this issue).

Reply:
yup the issue came from this commit in .data (when I updated region_isos to the WEO definitions):
https://github.com/2DegreesInvesting/r2dii.data/commit/f30a1fcb98977b55d81337e7bfd713ce6f68ac93

Initially `region_isos` was a made up fake dataset, and I guess i had used `oecd_europe` as a region, but in the actual weo_2019 no such region exists.

Reply:
By checking out locally to the previous commit (2DegreesInvesting/r2dii.data@b9d714c15633fb6024fd7a737e35c56ed336f0ea), I can get the following reprex: 
``` r
library(r2dii.data)
library(magrittr)

region_isos %>% dplyr::filter(region == "oecd_europe")
#> # A tibble: 27 x 2
#>    region      isos 
#>    <chr>       <chr>
#>  1 oecd_europe at   
#>  2 oecd_europe be   
#>  3 oecd_europe cz   
#>  4 oecd_europe dk   
#>  5 oecd_europe ee   
#>  6 oecd_europe fi   
#>  7 oecd_europe fr   
#>  8 oecd_europe de   
#>  9 oecd_europe gr   
#> 10 oecd_europe hu   
#> # … with 17 more rows
```

<sup>Created on 2020-05-28 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Good that the warning brought this to our attention!

Reply:
(Sorry, in short, the current behaviour is the desired behaviour. `oecd_europe` should not exist. just `oecd` should).